sfsu-csc-667-fall-2024-roberts/term-project-team
# fix: improve game initialization and player tokens

This PR improves the game initialization flow and player token handling:

Changes:
- Add proper delays between game messages and actions
- Improve player token visibility and positioning
- Add roll status updates in game header
- Fix bot roll integration
- Add visual feedback for current player's token
- Update player status messages

Testing:
1. Create a new game with a bot
2. Roll dice for initial turn order
3. Verify messages appear with proper timing
4. Verify bot rolls automatically after human
5. Verify player tokens are clearly visible
6. Verify status messages update correctly
